test/accel/CMakeLists.txt

```diff
@@ -100,5 +107,11 @@ celeritas_add_integration_tests(
   OFFLOAD cpu gpu g4
   RMTYPE ${_optical_rm_type}
 )
+celeritas_add_integration_tests(
+  TrackingManagerIntegration OpNoviceOptical run
+  OFFLOAD cpu gpu g4
+  RMTYPE ${_optical_rm_type}
+)
```

Comment:
Since the `OpNoviceOptical.run` tests are being added in the loop above, this is redundant.